### PR TITLE
Jam-setup.py -> bootstrap.js

### PR DESCRIPTION
**Pull request by chrisseto**

### CODE REVIEW

File: dev/generate_schemas.py [hunk 1]
```diff
@@ -1,240 +0,0 @@
-import json
```
Comment (samchrisinger) [inline dev/generate_schemas.py:1]: @chrisseto, I think we want this file. Can you please undelete it and move it to /scripts?

Comment (chrisseto) [inline dev/generate_schemas.py:1]: Converted to javascript as well?

Comment (samchrisinger) [inline dev/generate_schemas.py:1]: Only if you have time. Otherwise I'll do this post merge.

**Comment by samchrisinger** [comment]
ty, merging manually :car:

**Pull request closed without merging**